README.md

```diff
@@ -1 +1,72 @@
-# transport_optimum_client
\ No newline at end of file
+# Transport Optimum
```

Comment:
Clean this up, I would specify the purpose of this project here

Reply:
Done